webapp/src/components/training/TrainingInformation.vue

```diff
@@ -232,85 +250,72 @@ const lossSeries = computed(() =>
       },
     ),
 );
-const darkMode = localStorage.getItem("theme") === "dark";
-const commonChartsOptions = {
-  chart: {
-    toolbar: { show: false },
-    zoom: { enabled: false },
-  },
-  dataLabels: { enabled: false },
-  colors: [darkMode ? "#cbd5e1" : "#6096BA"],
-  fill: {
-    colors: [darkMode ? "#1A3A4F" : "#E2E8F0"],
-    type: "solid",
-    opacity: 0.6,
-  },
-  stroke: { curve: "monotoneCubic" },
-  grid: { show: false },
-  xaxis: {
-    type: "numeric",
-    decimalsInFloat: 0,
-    labels: {
-      show: true,
-      style: {
-        colors: darkMode ? "#ffffff" : "#000000",
-      },
-    },
-    axisBorder: {
-      show: true,
-      colors: darkMode ? "#ffffff" : "#000000",
-    },
-    axisTicks: {
-      show: true,
-      color: darkMode ? "#ffffff" : "#000000",
+
+const chartsLabels = computed<string[]>(() =>
+  Range(1, allEpochs.value.size + 1)
+    .map((e) => `Epoch ${e}`)
```

Comment:
```suggestion
    .map((e) => `${e}`)
```

Can we remove the "Epoch" prefix? I find the axis too crowded when there are many epochs